lightningd/options.c

```diff
@@ -222,7 +222,7 @@ static char *opt_add_addr_withtype(const char *arg,
 	assert(arg != NULL);
 	dns_ok = !ld->always_use_proxy && ld->config.use_dns;
 
-	/* Will be overridden in next call iff has port */
+	/* Will be overridden in next call, if it has a port */
```

Comment:
Hehe, this may look like a typo, but it is the scientific short-hand for `if-and-only-if`, so don't go around fixing it everywhere :-)
It's ok here ^^

Reply:
okay, but even then the work `a` was missing :D